codes/classical/q-ary_digits/simplex.yml

```diff
@@ -0,0 +1,27 @@
+code_id: simplex
+physical: q-ary_digits
+logical: q-ary_digits
+
+name: 'Simplex code'
+
+description: |
+  An \([n,k,q^{k-1}]\) linear code over \(\mathbb{F}_q\), where \(n=\frac{q^k-1}{q-1}\). The columns of the generator matrix are in one-to-one correspondence with the elements of the projective space \(PG(k-1,q)\), with each column being a chosen representative of the corresponding element.
+
+features:
+
+  decoders:
+    - 'A quantum decoding algorithm \cite{manual:{A. Barg and S. Zhou, A quantum decoding algorithm of the simplex code, 2004.}}.'
```

Comment:
The code can be decoded according to maximum likelihood because it is of small size. Faster decoders of the binary simplex codes are given in 
R. R. Green, "A serial orthogonal decoder," JPL Space Programs Summary, vol. 37–39-IV, pp. 247–253, 1966.
and 
Ashikhmin, Alexei [Ashikhmin, Alexei E.] (1-BELL);
Litsyn, Simon [Litsyn, Simon N.] (IL-TLAV-EN)
Simple MAP decoding of  first-order Reed-Muller and Hamming codes. (English
summary)
IEEE Trans. Inform. Theory 50 (2004), no. 8, 1812--1818.
The cited quantum algorithm also applies only to binary codes.